checkout module assignment/Merve/index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Ubuntu:ital,wght@0,300;0,400;0,500;0,700&display=swap" rel="stylesheet">
+    <title>Checkout Form</title>
+</head>
+<body>
+    <div class="container__main--parent">
+        <div class="container__main--leftchild">
+            <nav class="nav__list">
+                <ul>
+                    <li class="nav__list--item">
+                        <span>1</span>
+                        <div>
+                            <p>STEP 1</p>
+                            <a href="#">YOUR INFO</a>
```

Comment:
In a page like this, you'd need JS to make the navigation works so I'd not use <a> element here (just a <span>) and make entire <li> clickable so the user does not have to figure out what to click on to go to another section.